keps/sig-windows/4885-windows-cpu-and-memory-affinity/README.md

```diff
@@ -0,0 +1,594 @@
+
+##### Prerequisite testing updates
+
+##### Unit tests
+
+- pkg/kubelet/cm/container_manager_windows.go
+- pkg/kubelet/cm/internal_container_lifecycle_windows.go
+- pkg/kubelet/winstats/cpu_topology_test.go
+
+##### Integration tests
+
+Integration tests do not run on Windows. Functionality will be covered by unit and e2e tests.
+
+##### e2e tests
+
+-  e2e_node will need to be enabled for windows to add coverage
+
+### Graduation Criteria
+
+#### Alpha
+
+- Feature implemented behind a feature flag
+- Initial basic e2e tests in Windows e2e suite are added
+- unit tests for Windows specific components are added
+
+#### Beta
+
+- Gather feedback from developers 
+- e2e_node tests are in Testgrid and linked in KEP
+
+#### GA
+
+- 2 examples of real-world usage
+- Allowing time for feedback
+
+**Note:** Generally we also wait at least two releases between beta and
+GA/stable, because there's no opportunity for user feedback, or even bug reports,
+in back-to-back releases.
+
+**For non-optional features moving to GA, the graduation criteria must include
+[conformance tests].**
+
+[conformance tests]: https://git.k8s.io/community/contributors/devel/sig-architecture/conformance-tests.md
+
+#### Deprecation
+
+N/A
+
+### Upgrade / Downgrade Strategy
+
+### Version Skew Strategy
+
+N/A
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+<!--
+Pick one of these and delete the rest.
+
+Documentation is available on [feature gate lifecycle] and expectations, as
+well as the [existing list] of feature gates.
+
+[feature gate lifecycle]: https://git.k8s.io/community/contributors/devel/sig-architecture/feature-gates.md
+[existing list]: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
+-->
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: WindowsCPUAndMemoryAffinity
+  - Components depending on the feature gate:
+- [ ] Other
+  - Describe the mechanism:
+  - Will enabling / disabling the feature require downtime of the control
+    plane?
+    No
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node?
+    Yes it uses a feature gate. Memory and CPU managers have a state file that requires cleanup.
```

Comment:
Describe the steps explicitly.  Do kubelets need to be drained?

Reply:
Add some details and linked to existing keps and docs where this is covered